services/metadata/frontend/Menu.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+  import BurgerButton from "./BurgerButton.svelte";
+
+  let visible = "hidden";
+
+  function toggleVisible() {
+    if (visible == "hidden") {
+      visible = "visible"
+    } else if (visible == "visible") {
+      visible = "hidden"
+    }
+  }
```

Comment:
With boolean it's one-liner: `isVisible = !isVisible"

Reply:
see comment above

Reply:
With my above comment this function needs to return boolean, so:
```
function toggleVisible(): boolean {
  return isVisible = !isVisible;
}